modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/PlayerListener.java

```diff
@@ -192,15 +193,16 @@ public void onPlayerMove(PlayerMoveEvent event) {
         }
 
         if (c.isNotProcessing() && c.getType().getMoveEntities() && !timeToReleaseAfter.containsKey(c)) {
-            if (Settings.ManOverBoardTimeout != 0) {
-                event.getPlayer().sendMessage(I18nSupport.getInternationalisedString("You have left your craft. You may return to your craft by typing /manoverboard any time before the timeout expires"));
+            if (Settings.ManOverboardTimeout != 0) {
+                p.sendMessage(I18nSupport.getInternationalisedString("You have left your craft. You may return to your craft by typing /manoverboard any time before the timeout expires"));
+                CraftManager.getInstance().addOverboard(p, c);
```

Comment:
this will only update if the player is the pilot because of how you get the craft